### ISSUE TITLE
Remove spaces from database view column names

### ISSUE THREAD

**Issue by NiccyB** [hypercrypt/octopus-db#3]
The database views are great for reporting on, but I wonder if the column names with spaces in them should be renamed to remove the spaces.

**Comment by hypercrypt** [comment]
Happy to. Would you be happy to suggest column names?

**Comment by NiccyB** [comment]
How's about:

`off-peak kWh` --> `offpeak_kWh`
`peak kWh` --> `peak_kWh`
`total kWh` --> `total_kWh`
`p/kWh` --> `pp_kWh`

**Comment by hypercrypt** [comment]
> Hi You probably meant this to go elsewhere. Not to worry!! :) >

You'll be getting these emails as you're watching the repo. If you don't want to get them then you may want to tweak your settings or un-watch and start instead. Your feedback is always welcome though

**Comment by hypercrypt** [comment]
I propose this as the updated structure for the half-hour view:

```
CREATE VIEW `billing_e_hh` AS
SELECT
   FROM_UNIXTIME(`u`.`interval_start`)                                                      AS `period`,
   ROUND(`u`.`consumption`, 3)                                                              AS `kwh_total`,
   ROUND(IF((`u`.`reduced_rate_inc_vat` >  `t`.`offpeakThreshold`), `u`.`consumption`,0),3) AS `kwh_peak`,
   ROUND(IF((`u`.`reduced_rate_inc_vat` <= `t`.`offpeakThreshold`), `u`.`consumption`,0),3) AS `kWh_offpeak`,
   ROUND((`u`.`reduced_price_inc_vat` / 100),6)                                             AS `cost`,
   ROUND(`u`.`reduced_rate_inc_vat`,4)                                                      AS `ppkwh`,
   `u`.`tariff`                                                                             AS `tariffs`
FROM `usage` `u`
	LEFT JOIN `tariff` `t`
		ON `u`.`tariff` = `t`.`tariff`
		AND `u`.`interval_start` >= `t`.`valid_from`
		AND `u`.`interval_end`   <= `t`.`valid_to`
WHERE (`u`.`fuel` = 'electricity')
ORDER BY `u`.`interval_start` DESC;
```

The views covering more than half hour would also have a `percent_offpeak` column

**Comment by hypercrypt** [comment]
This is fixed in 0.0.3. Use `php octo db/views` to get the new views
